packages/cxl-ui/src/components/cxl-jw-player/mixins/index.js

```diff
@@ -0,0 +1,5 @@
+export { BaseMixin } from './BaseMixin';
+export { ChapterNavigationMixin } from './chapter-navigation/index.js';
+export { NextUpMixin } from './NextUpMixin';
+export { StateMixin } from './StateMixin';
+export { TranscriptMixin } from './TranscriptMixin';
```

Comment:
"Kebab-case everywhere" possible?

Maybe we can also install https://github.com/loeffel-io/ls-lint one day.

Reply:
I'm trying to remember the reason for using pascal case vs. kebab case for mixins. I'll take a look and find a standard.